subscription-service/src/test/kotlin/com/egm/stellio/subscription/job/TimeIntervalNotificationJobTest.kt

```diff
@@ -89,33 +89,33 @@ class TimeIntervalNotificationJobTest {
             EntityInfo(
                 id = "urn:ngsi-ld:FishContainment:1234567890".toUri(),
                 idPattern = ".*FishContainment.*",
-                type = "FishContainment"
+                type = "https://uri.fiware.org/ns/data-models#FishContainment"
             )
         )
         val q = "speed>50;foodName==dietary fibres"
 
         assertEquals(
             "?type=FishContainment" +
                 "&id=urn:ngsi-ld:FishContainment:1234567890" +
-                "&q=speed>50;foodName==dietary fibres",
+                "&q=speed%3E50%3BfoodName%3D%3Ddietary+fibres",
```

Comment:
should not it be `speed%3E50%3BfoodName%3D%3Ddietary%20fibres`?

Reply:
ah... so what was the failed test about?

Reply:
The test returns `Request was not matched`. I have this request for the stub `/ngsi-ld/v1/entities?type=BeeHive&q=speed%3E50%3BfoodName%3D%3Ddietary+fibres` and my request is `/ngsi-ld/v1/entities?type=BeeHive&q=speed%253E50%253BfoodName%253D%253Ddietary+fibres`

Reply:
there is a double URL-encoding going on

Reply:
Yes when I make my request with `webClient`. It encodes also the url.